ash/src/device.rs

```diff
@@ -2008,6 +2008,29 @@ pub trait DeviceV1_0 {
             .get_render_area_granularity(self.handle(), render_pass, &mut granularity);
         granularity
     }
+
+    #[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkGetDeviceMemoryCommitment.html>"]
+    unsafe fn get_device_memory_commitment(
+        &self,
+        memory: vk::DeviceMemory,
+    ) -> vk::DeviceSize {
+        let mut committed_memory_in_bytes = mem::zeroed();
```

Comment:
the mem::zeroed() seems unnecessary ?

Reply:
Yes, but does it really hurt? It is consistent with other functions that return more elaborate types and allows for boiler-plate code that is return type agnostic.

Reply:
Other functions using "more elaborate types" have actual _elaborate_ structs with multiple fields. Zero-initialization [is not always correct there either](https://github.com/MaikKlein/ash/pull/394) and I'd _personally_ prefer to just use `Default::default()` everywhere.

In this case however `vk::DeviceSize` is not elaborate at all; it is merely a typedef to `u64` (a 64-bit device pointer). Why don't we treat it that way and make it clear from the get-go, just use `0` like everywhere.

Reply:
@alpacahaircut Can you update this with `0` so that we can merge this PR?